radicle-cli/src/commands/node.rs

```diff
@@ -0,0 +1,165 @@
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+use radicle::node::{Address, Node, NodeId};
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+#[path = "node/control.rs"]
+mod control;
+#[path = "node/routing.rs"]
+mod routing;
+#[path = "node/status.rs"]
+mod status;
+#[path = "node/tracking.rs"]
+mod tracking;
+
+pub const HELP: Help = Help {
+    name: "node",
+    description: "Control and query the Radicle Node",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad node status
+    rad node (start|stop)
+    rad node connect <NID> <ADDR>
```

Comment:
Do we capitalise these or have them lowercase?

Reply:
Lowercase :)